provisioner/ansible/provisioner.go

```diff
@@ -382,6 +391,21 @@ func validateFileConfig(name string, config string, req bool) error {
 	return nil
 }
 
+func validateDirectoryConfig(name string, config string, req bool) error {
```

Comment:
I got rid of `req`, but `config` is used in the errors that `validateDirectoryConfig` returns, so I left it in.  Let me know if I should change the error message so that this function doesn't need the `config` param.